packages/terminal/src/browser/terminal-frontend-contribution.ts

```diff
@@ -230,6 +251,10 @@ export class TerminalFrontendContribution implements TerminalService, CommandCon
             keybinding: 'ctrlcmd+k',
             context: TerminalKeybindingContexts.terminalActive
         });
+        keybindings.registerKeybinding({
+            command: TerminalCommands.SHOW_ALL_OPENED_TERMINALS.id,
+            keybinding: 'ctrlcmd+k ctrlcmd+t'
```

Comment:
I'm not sure this is the correct keybinding.
<kbd>ctrlcmd</kbd>+<kbd>k</kbd> <kbd>ctrlcmd</kbd>+<kbd>t</kbd> is used to change theme in vscode.

Reply:
Sure! Just removed the keybinding.